dist/angular-dashboard-framework.css

```diff
@@ -44,6 +44,17 @@
   width: 98%;
 }
 
+/*
+Accessibility
+ */
+/*
+buttons without borders
+ */
+.btn-wo-border {
```

Comment:
This change is not persistent. When running `gulp all` it will be overriden.

Reply:
These changes should be discarded as they are already defined in the `src/styles/main.scss`